ampd/src/block_height_monitor.rs

```diff
@@ -47,8 +52,8 @@ impl<T: TmClient + Sync> BlockHeightMonitor<T> {
         loop {
             select! {
                 _ = interval.tick() => {
-                    let latest_block = self.client.latest_block().await.change_context(BlockHeightMonitorError::LatestBlockError)?;
-                    self.latest_height.store(latest_block.block.header.height.into(), std::sync::atomic::Ordering::SeqCst);
+                    let latest_block = self.client.latest_block().await.change_context(BlockHeightMonitorError::LatestBlock)?;
+                    self.latest_height_tx.send(latest_block.block.header.height.into()).expect("failed to publish latest block height");
```

Comment:
Can you add a comment that the `expect` is fine because the receiver `latest_height_rx` is never closed and therefore the channel stays open?